docs/docs/50-user-guide/60-reference-docs/80-webhook-receivers/github.md

```diff
@@ -0,0 +1,190 @@
+---
+sidebar_label: GitHub
+---
+
+# GitHub
+
+## Required Secrets for Github
+
+Our webhook receiver will require a Kubernetes secret. This secret is required to contain a `secret` key in its `stringData`. We will need to provide the value we assigned to the `secret` key to GitHub in a later step so keep it handy.
```

Comment:
```suggestion
The Kargo GitHub webhook receiver will require a Kubernetes secret. This secret is required to contain a `secret` key in its `stringData`. We will need to provide the value we assigned to the `secret` key to GitHub in a later step so keep it handy.
```

Reply:
Note that applies throughout. When referring to a resource of type `Secret`, be sure to capitalize and monospace. Especially in this context, observing that typographical convention helps readers to distinguish between the _concept_ of a closely guarded value and an actual `Secret` resource.